### ISSUE TITLE
Provide more specific guidance on the Bundle needed for draftOrders

### ISSUE THREAD

**Issue by cooperthompson** [cds-hooks/docs#517]
The [order-select](https://cds-hooks.org/hooks/order-select/) and [order-sign](https://cds-hooks.org/hooks/order-sign/) hooks both specify that a FHIR Bundle should be sent in the draftOrers field.  The [Bundle ](https://www.hl7.org/fhir/bundle.html#resource)resource requires a type, but the CDS Hooks spec doesn't specify what the type of the draftOrders Bundle should be.  I think collection makes the most sense.  

Regardless, it would be nice to have a little more guidance on exactly what the draftOrders Bundle should look like.

**Comment by jmandel** [comment]
Agreed on `Bundle.type` of `collection`. Any other details you think we should lock down here?

**Comment by cooperthompson** [comment]
Bundle.type was the main property that jumped out.  Everything else is optional and seems fine to leave under-specified.
